Source/Blazorise.Bulma/BulmaClassProvider.cs

```diff
@@ -793,10 +793,20 @@ public override string RowColumns( RowColumnsSize rowColumnsSize, RowColumnsDefi
 
     #region Column
 
-    public override string Column( bool hasSizes ) => hasSizes ? null : "column";
+    public override string Column( bool grid, bool hasSizes ) => hasSizes ? null : "column";
 
-    public override string Column( ColumnWidth columnWidth, Breakpoint breakpoint, bool offset )
+    public override string Column( bool grid, ColumnWidth columnWidth, Breakpoint breakpoint, bool offset )
     {
+        if ( grid )
+        {
+            if ( breakpoint != Blazorise.Breakpoint.None )
+            {
+                return $"is-grid-column-{ToColumnWidth( columnWidth )}-{ToBreakpoint( breakpoint )}";

```

Comment:
What's the difference between **is** and **are**?

Reply:
That's usually the Bulma naming convention. `is-` is meant for the single element, and `are-` is meant for multiple elements, usually child elements.